src/Components/Address/AddressFormFields.tsx

```diff
@@ -252,6 +283,57 @@ export const AddressFormFields = <V extends FormikContextWithAddress>(
           />
         </Column>
       )}
+      {phoneInputType === "rich" && (
+        <Column span={12}>
+          <PhoneInput
+            name="phoneNumber"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            data-testid={`${dataTestIdPrefix}.phoneNumber`}
+            options={countryPhoneOptions}
+            onSelect={(option: CountryData): void => {
+              setFieldValue("phoneNumberCountryCode", option.countryCode)
+            }}
+            dropdownValue={values.phoneNumberCountryCode}
+            inputValue={values.phoneNumber}
+            placeholder="(000) 000 0000"
+            error={
+              (touched.phoneNumberCountryCode &&
+                (errors.phoneNumberCountryCode as string | undefined)) ||
+              (touched.phoneNumber &&
+                (errors.phoneNumber as string | undefined))
```

Comment:
We shouldn't need the `as string | undefined`

Reply:
I think this requires a change in palette - without it we get Type 'string[]' is not assignable to type 'string | boolean | undefined'

Reply:
ah weird -- ok all good 👍